tn_api_hotel.yml

```diff
@@ -239,89 +312,449 @@ components:
+                      minimum_charge:
+                          type: float
+                          description: The minimun total for the requested search available (in the currency provided in the search request)
+                          example: 400.32
+                      maximum_charge:
+                          type: float
+                          description: The maximum total priced room option available (in the currency provided in the search request)
+                          example: 780.65
+                      address:
+                          type: object
+                          properties:
+                              street_address:
+                                  type: string
+                                  description: The street address of the hotel.
+                                  example: '1583 Brunswick St, Halifax, NS B3J 3P5'
+                              longitude:
+                                  type: float
+                                  description: The longitude of the hotel.
+                                  example: 44.64511447836071
+                              lattitude:
+                                  type: float
+                                  description: The lattitude of the hotel.
+                                  example: -63.575245615440565
+                              distance_from_city_km:
+                                  type: float
+                                  description: The distance that the hotel is from the "City Centre".
+                                  example: 7.5
+                              phone_number:
+                                  type: str
+                                  description: The contact phone number for the hotel.
+                                  example: '+1 902-420-0555'
+                      ammenities:
+                        type: array
+                        description: A list of the amenities provided at this hotel.
+                        items:
+                          type: object
+                          properties:
+                            amenity_type:
+                              type: string
+                              description: The amenity type, can be either HA (Hotel Amenity) or RA (Room amenity)
+                              example: "HA"
+                            amenity_title:
+                              type: string
+                              description: A title or descriptor of the amenity provided.
+                              example: High speed internet connection
+          hotel_itinerary_reference:
+              type: string
+              example: '1'
+              description: A reference that indicates the itineraries position in the flow of itineraries in the super_trip with the index starting at 0.
 
     HotelSearchRequest:
-      required:
-        - super_trip_id
       properties:
```

Comment:
Add an optional NextResultReference, this is returned in a previous search result for the same trip. This is used in case you want to scroll down and load more results for the same search.